lib/mockbuilder.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
The way I've done it, only the packages explicitly passed would be built, so passing `gcc, kernel` would only build gcc once. I'm applying your other suggestions though, so this will be redone.

I don't get your second point, doesn't your suggestion of using a primary key lead to package (and commit) specific directories?